archiver/direct.py

```diff
@@ -30,24 +37,33 @@ def __archive(self, submission: HcaSubmission):
             biosamples = self.__biosamples_submitter.send_all_samples(submission)
             ingest_entities_to_update.extend(biosamples.get('CREATED', []))
         if self.__biostudies_submitter:
-            raise NotImplementedError("BioStudies Submitter not implemented")
+            biostudies = self.__biostudies_submitter.send_all_projects(submission)
+            ingest_entities_to_update.extend(biostudies.get('CREATED', []))
         for entity in ingest_entities_to_update:
             submission.add_accessions_to_attributes(entity)
             self.__updater.update_entity(entity)
 
 
 def direct_archiver_from_params(
         ingest_url: str,
-        biosamples_url: str, biosamples_domain: str, aap_url: str, aap_user: str, aap_password: str
+        biosamples_url: str, biosamples_domain: str, aap_url: str, aap_user: str, aap_password: str,
```

Comment:
I would use a parameter object here instead of the many arguments.

Reply:
@amnonkhen I would agree with you in another case, but if you read the code carefully and look at the caller we unpack a parameter object by this way. This way is a bit shorter than to unpack everything from the parameter object line by line. What do you think?